Components/Sensors/BNO055.cs

```diff
@@ -0,0 +1,333 @@
+﻿using System;
+using System.Threading;
+using Scarlet.Components;
+using Scarlet.IO;
+
+namespace Scarlet.Components.Sensors
+{
+    public class BNO055 : ISensor
+    {
+        public const byte BNO055_ADDRESS_A = 0x28;
+        public const byte BNO055_ADDRESS_B = 0x29;
+        public const byte BNO055_ID = 0xA0;
+        public const byte NUM_BNO055_OFFSET_REGISTERS = 22;
+
+        /// <summary>
+        /// Enum of all possible registers on the BNO055. 
+        /// </summary>
+        private enum Register : byte
+        {

```

Comment:
Do you think you could go through and give these more friendly, descriptive names? Right now this list is essentially useless unless you already have code set up.

Reply:
That's a really big list, and Nathan says I should focus on writing the CAN implementation.